### PR TITLE
submit clearblade helm charts

### PR DESCRIPTION
**Pull request by ryanmccluretx**

### CODE REVIEW

File: applications/clearblade-edge/clearbladehelm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```
Comment (abjyoti) [inline applications/clearblade-edge/clearbladehelm/Chart.yaml:1]: Please add Copyright header with License type

File: applications/clearblade-edge/clearbladehelm/templates/pod-intel.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```
Comment (abjyoti) [inline applications/clearblade-edge/clearbladehelm/templates/pod-intel.yaml:1]: Please add Copyright header with License type

File: applications/clearblade-edge/clearbladehelm/templates/pod-intel.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+kind: Pod
```
Comment (abjyoti) [inline applications/clearblade-edge/clearbladehelm/templates/pod-intel.yaml:2]: Please add Copyright header with License type

File: applications/clearblade-edge/clearbladehelm/templates/service.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```
Comment (abjyoti) [inline applications/clearblade-edge/clearbladehelm/templates/service.yaml:1]: Please add Copyright header with License type

File: applications/clearblade-edge/clearbladehelm/templates/volume-claim-intel.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```
Comment (abjyoti) [inline applications/clearblade-edge/clearbladehelm/templates/volume-claim-intel.yaml:1]: Please add Copyright header with License type

File: applications/clearblade-edge/clearbladehelm/templates/volume-intel.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```
Comment (abjyoti) [inline applications/clearblade-edge/clearbladehelm/templates/volume-intel.yaml:1]: Please add Copyright header with License type

File: applications/clearblade-edge/clearbladehelm/values.yaml [hunk 7]
```diff
@@ -0,0 +1,21 @@
+replicaCount: 1
```
Comment (abjyoti) [inline applications/clearblade-edge/clearbladehelm/values.yaml:1]: Please add Copyright header with License type

**Review by abjyoti — COMMENTED** [review]
Please add Copyright headers with License info

**Review by abjyoti — CHANGES_REQUESTED** [review]
Please provide the README.md with details on Getting started/ User Guide information.

File: applications/clearblade-edge/clearbladehelm/values.yaml [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+image:
+  name: clearblade/edge:9.7.0
+  pullPolicy: IfNotPresent
+  #tag: "latest"
```
Comment (amr-mokhtar) [inline applications/clearblade-edge/clearbladehelm/values.yaml:6]: Is the tag needed? Should it be `9.7.0` ?

File: applications/clearblade-edge/clearbladehelm/templates/service.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+  name: clearblade-edge2
+  labels:
+    app: clearblade-edge2
+spec:
+  type: {{ .Values.serviceType }}
+  ports:
+    - port: 9000
+      protocol: TCP
+      targetPort: 9000
+      #name: sql
```
Comment (amr-mokhtar) [inline applications/clearblade-edge/clearbladehelm/templates/service.yaml:13]: Is this commented line is still needed?

**Comment by amr-mokhtar** [comment]
> Please provide the README.md with details on Getting started/ User Guide information.

Consider the users who came across this GitHub repo and would like to get started with the Clearblade Edge platform.

**Comment by BoguInt** [comment]
According to https://helm.sh/docs/chart_best_practices/conventions/:

> The directory that contains a chart MUST have the same name as the chart

**Comment by ryanmccluretx** [comment]
Added copyrights and readme, got rid of comments in yaml

**Review by abjyoti — APPROVED** [review]
I am fine with resolutions

**Review by amr-mokhtar — APPROVED** [review]
Thanks Clearblade team for the contribution. Welcome to the OpenNESS project!!

**Pull request merged**